go/governance/gen_vectors/main.go

```diff
@@ -15,9 +16,30 @@ import (
 	"github.com/oasisprotocol/oasis-core/go/consensus/api/transaction"
 	"github.com/oasisprotocol/oasis-core/go/consensus/api/transaction/testvectors"
 	governance "github.com/oasisprotocol/oasis-core/go/governance/api"
-	"github.com/oasisprotocol/oasis-core/go/upgrade/api"
+	upgrade "github.com/oasisprotocol/oasis-core/go/upgrade/api"
 )
 
+func validSubmitProposalTxFields(v uint16, epoch uint64, handler string, target version.ProtocolVersions) bool {
```

Comment:
Why not just call `ValidateBasic`?

Reply:
There could be a validation helper function for each transaction type. Currently, there are 2.

I could rename it to `validateSubmitProposalBasic` or `validateBasicSubmitProposal` or just `validteSubmitProposal`?

And the second one accordingly.

Reply:
I've changed all helper names to use the last variant, e.g. `validateSubmitProposal`.